tests/test_abort.js

```diff
@@ -57,3 +57,28 @@ test("abort is emitted before delay time", function(t) {
     req.abort();
   }, 10);
 });
+
+test("nock#882: Aborting an aborted request should not emit an error", function(t) {
+  nock('http://test.example.com')
+    .get('/status')
+    .reply(200);
+
+  var tstart = Date.now();
+  var errorCount = 0
+  var req = http.get('http://test.example.com/status')
+    .on('error', function(err) {
+      errorCount++;
+      if (errorCount < 3) {
+        // Abort 3 times at max, otherwise this would be an endless loop,
+        // if #882 pops up again.
+        req.abort();
+      }
+    });
+  req.abort();
+
+  // Give nock some time to fail
+  setTimeout(function() {
+    t.equal(errorCount, 1,"Only one error should be sent.");
+    t.end();
+  },10);
```

Comment:
I tried that and got a `test unfinished`. It seems like the 'end'-event is not in this case. I also tried piping the response to a writable stream to make sure that the response is 

Is an `end`-event supposed to be sent, when a request is aborted? But if "yes", then it would certainly be sent after the first abort, and not  after the second one. In that case, the test could pass, even if the issue wasn't fixed, because it could happen that the `t.equal`-check is evaluated too early.

I think it is safer to use 'setTimeout` in this case.